### ISSUE TITLE
`eslint-comments/no-unused-disable` false positive when disabling line with `eslint-comments/no-unused-disable`

### ISSUE THREAD

**Issue by DetachHead** [mysticatea/eslint-plugin-eslint-comments#77]
```ts
// eslint-disable-next-line eslint-comments/no-unlimited-disable
// eslint-disable-next-line
```
errors:
```
'eslint-comments/no-unlimited-disable' rule is disabled but never reported.eslint (eslint-comments/no-unused-disable)
Unexpected unlimited 'eslint-disable-next-line' comment. Specify some rule names to disable. (eslint-comments/no-unlimited-disable)
```

**Comment by MichaelDeBoey** [comment]
Hi @DetachHead!

Since this repo is unmaintained, you might want to re-open this issue in the @eslint-community fork https://github.com/eslint-community/eslint-plugin-eslint-comments

For more info about why we created this organization, you can read https://eslint.org/blog/2023/03/announcing-eslint-community-org

**Comment by DetachHead** [comment]
@MichaelDeBoey thanks. i've re-raised it https://github.com/eslint-community/eslint-plugin-eslint-comments/issues/130

btw you may be interested in https://github.com/DetachHead/issues-transfer if you want to automatically transfer all the issues from this repo

**Comment by MichaelDeBoey** [comment]
@DetachHead Unfortunately, I don't have any maintainer-rights on this repo

If I would, we would have transferred the whole repo to the community org and get rid of the fork 😛

**Comment by DetachHead** [comment]
i don't think you need maintainer rights, since all it does is copy the contents of the open issues and re-raises them on the specified repo. i don't think you need maintainer access on either repo so if you want i'm happy to run it myself

**Comment by MichaelDeBoey** [comment]
@DetachHead I think almost all issues have one in the fork, so it's fine
